tools/find-files.ini

```diff
@@ -0,0 +1,32 @@
+[all]
+\[a-z]*
+.gitattributes
+.gitignore
+
+[ignore]
+components/cjson
+components/esptool_py
+components/lwip
+components/ssl/axtls
+components/ssl/mbedtls
+components/ssl/wolfssl
+tools/kconfig
+tools/kconfig_new
+*.bin
+*.o
+*.a
+*.mk
+*.key
+*.rar
+sdkconfig
+sdkconfig.old
+
+[no-expand-tabs]
+*.mk
+Makefile*
+makefile*
```

Comment:
Makefile.projbuild is matched by Makefile* so that should be covered.

Checked with `tools/find-files | grep Makefile.projbuild`.

Reply:
None of the *.cmake files use tabs today except ./tools/cmake/third_party/GetGitRevisionDescription.cmake.

Should they be switched to use tabs instead?

Reply:
`no-expand-tabs` means that don't use spaces to replace tab?

```
$ tools/find-files | grep Makefile.projbuild
components/esp8266/Makefile.projbuild	ignore=False	no_expand_tabs=True	clang_format=False
```
```
$ tools/reformat -e
Replacing tabs with spaces...
...
Fixed components/esp8266/Makefile.projbuild
```
The tab in `components/esp8266/Makefile.projbuild` will be replaced, it will break the make.

Reply:
Which version of pathspec do you have? I have 0.5.6.

Reply:
@27-301 Can you check your version?